.template-helpers/variant.md

```diff
@@ -0,0 +1,7 @@
+# Image Variants
+
+The `%%REPO%%` images come in many flavors, each designed for a specific use case.
```

Comment:
Hmm, we might actually already have examples of this being "something something `%%REPO%%` image" (singular); requires more thought.